lib/schema.js

```diff
@@ -38,7 +38,8 @@ var express = require("express"),
     RemoteAccessToken = require("./model/remoteaccesstoken").RemoteAccessToken,
     RemoteRequestToken = require("./model/remoterequesttoken").RemoteRequestToken,
     ActivityObject = require("./model/activityobject").ActivityObject,
-    DatabankStore = require('connect-databank')(express),
+    session = require("express-session"),
+    DatabankStore = require('connect-databank')(session),
```

Comment:
I don't quite understand this change - can you explain why this is needed?

Also, please use double quotes for consistency.